docs/changehistory/NextVersion.md

```diff
@@ -95,6 +98,31 @@ Previously, iTwin.js used [DELETE](https://www.sqlite.org/pragma.html#pragma_jou
 - Failure to close a writeable briefcase may leave a "-wal" file. Previously, if a program crashed or exited with an open briefcase, it would leave the briefcase file as-of its last call to `IModelDb.saveChanges`. Now, there will be another file with the name of the briefcase with "-wal" appended. This is not a problem and the briefcase is completely intact, except that the briefcase file itself is not sufficient for copying (it will not include recent changes.) The "-wal" file will be used by future connections and will be deleted the next time the briefcase is successfully closed.
 - Attempting to copy an open-for-write briefcase file may not include recent changes. This scenario generally only arises for tests. If you wish to copy an open-for-write briefcase file, you must now call [IModelDb.performCheckpoint]($backend) first.
 
+## Presentation
+
+### Tree hierarchy levels filtering
+
+Ability to filter individual hierarchy level in tree was added for the trees that uses [PresentationTreeDataProvider]($presentation-components). To enable this [PresentationTreeRenderer]($presentation-components) should be passed to [ControlledTree]($components-react) through [ControlledTreeProps.treeRenderer]($components-react):
+
+```ts
+return <ControlledTree
+  // other props
+  treeRenderer={(treeProps) => <PresentationTreeRenderer {...treeProps} imodel={props.imodel} modelSource={filteredModelSource} />}
```

Comment:
Props aren't very clear (e.g. what is `filteredModelSource`?). Could we just pass `imodel={imodel} modelSoure={modelSource}`?

Reply:
Copied it straight from test-app did not notices it used those variables. Simplified.